src/xdocs/config_metrics.xml

```diff
@@ -101,6 +101,22 @@
         <source>
 &lt;module name=&quot;BooleanExpressionComplexity&quot;/&gt;
         </source>
+        <p>Code Example:</p>
+        <div class="wrapper">
+          <pre>
+            public class Test
+            {
+              public static void main(String ... args)
+              {
+                boolean a = true;
+                boolean b = false;
+
+                boolean c = (a &amp; b) | (b &amp; a); //OK
+                boolean d = (!a &amp;&amp; b) ^ (a || !b) &amp;&amp; a; //violation,complexity is 4
```

Comment:
Please explain on a subsequent line where the counts come from. It helps to explain to users how counting happens so they grasp the check easier.

Example: `X, Y, and Z are counted as 1 each which gives the total of 3. A, B, and C are ignored which don't add to the score`.
I would even consider adding counting to non-violations to show it is below the threshold.

Reply:
done